app/components/windows/SourcesShowcase.vue

```diff
@@ -64,6 +64,15 @@
       </ul>
```

Comment:
Really nitpicky, but since this is used below as well, might make sense to extract method on this one.